output/watermark-to-event.json

```diff
@@ -2,9 +2,9 @@
   "/common/destiny2_content/icons/ad7fdb049d430c1fac1d20cf39059702.png": 1,
   "/common/destiny2_content/icons/04de56db6d59127239ed51e82d16c06c.png": 2,
   "/common/destiny2_content/icons/52523b49e5965f6f33ab86710215c676.png": 3,
+  "/common/destiny2_content/icons/3543d23d9063fbf7332c7f129a74ada2.png": 4,
   "/common/destiny2_content/icons/f80e39c767f309f0b2be625dae0e3744.png": 4,
   "/common/destiny2_content/icons/65097e226318b8581ad535b33827e01a.png": 5,
   "/common/destiny2_content/icons/8b0d9b848bfb49077fe018e6f80a2939.png": 5,
-  "/common/destiny2_content/icons/3543d23d9063fbf7332c7f129a74ada2.png": 6,
```

Comment:
seems wrong

Reply:
This seems to be an empty watermark that's used as the "shelved"/sunset watermark for both FotL and Guardian Games items... seems fine to me.